doc/install-standalone.md

```diff
@@ -70,9 +70,7 @@ find . -name '*.php' -type f -print0 | xargs -0 sed -i "s#DemoConnector#${CONNEC
 At this point, you can clean all the starter kit initialization files:
 
 ```
-rm -f doc/*
-rm -rf vendor
-rm -rf .git
+rm -rf bin doc vendor .git
```

Comment:
I would prefer `rm -rf bin doc .git vendor/*`

The goal is to keep the `doc` folder to remind that documentation is very important (if not mandatory).

Thoughts ?